### PR TITLE
Grammar improvements in documentation

### PR DESCRIPTION
**Pull request by drishyasankar**
**Issue:**

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json`
  - [ ] I optimized the icon with SVGO or SVGOMG
  - [ ] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 </a>
 <h3 align="center">Simple Icons</h3>
 <p align="center">
-Free SVG icons for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
+Free SVG icons for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech.</a>.</p>
```
Comment (ericcornelissen) [inline README.md]: Actually, there is a dot after the `<a>` tag is closed, could you revert this change?

```suggestion
Free SVG icons for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
```

File: scripts/build-package.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 /**
  * @fileoverview
- * Compiles our icons into static .js files that can be imported in the browser
+ * Compiles our icons into static .js files that can be imported in the browser.
  * and are tree-shakeable.
```
Comment (ericcornelissen) [inline scripts/build-package.js]: Actually, the full sentence is as below, it just continues on the next line.

> Compiles our icons into static .js files that can be imported in the browser and are tree-shakeable.

**Comment by birjolaxew** [comment]
Given the nature of the PR, and the other activity on your account, I'll be closing this off as invalid for now. If you want to try your hand at contributing, please have a look at some of the issues in the repo and our [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon), and feel free to ask if you have any questions!
